homework2/task4.py

```diff
@@ -0,0 +1,16 @@
+# Задайте список из N элементов, заполненных числами из промежутка [-N, N].
+# Найдите произведение элементов на указанных позициях. Позиции хранятся в файле file.txt в одной строке одно число.
+
+n = int(input('Введите положительное число N '))
+
+list = 45, 20, 4, 92, 10, 5, 79, 39, 63, 67, 2, 52, 83, 59, 18, 41, 11, 22, 90, 3
```

Comment:
я подозреваю, что здесь нужно было вытаскивать список из вашего файла, но я не знаю, как это делать (вторую лекцию и второй семинар еще не смотрела), поэтому попыталась решить задачу вот так.
и все равно ничего не получилось :(

Reply:
Но тут как раз и нужно было посмотреть следующую лекцию, где рассказывается, как работать с файлами.
Например, можно сделать так:
```suggestion
lst = [int(line) for line in open('file.txt')]
```
И в качестве имени переменной не стоит брать имя типа данных, который эта переменная содержит.